### ISSUE TITLE
Unable to trade marks of honor

### ISSUE THREAD

**Issue by BrockSamson1** [thebarrensissues/Issues#1261]
**Description:**

A vendor was created next to Nolin for the purpose of providing access to AV marks of honor and thereby the honor gear that is gated by AV marks as a currency.
Unable to trade marks of honor for AV marks with the battleground token vendor

**Current behaviour:**

Approaching vendor with horde WSG marks in inventory gives the dialogue option to trade for AV tokens. Using this dialogue results in no exchange.

**Expected behaviour:**

Approaching vendor with horde WSG marks in inventory gives the dialogue option to trade for AV tokens. Using this dialogue results a trade of x warsong gulch marks for f(x) alterac valley marks. 

**Steps to reproduce the problem:**

CHANGEME
1. Step 1 Approach vendor with WSG marks
2. Step 2 Attempt to use dialogue option
3. Step 3 Observe no exchange

**Comment by giveusmezpls** [comment]
Hi Brock,
try separating them into a singular unit BEFORE clicking on the NPC, then click and use the gossip. let me know what the outcome is.

**Comment by BrockSamson1** [comment]
Welp - just checked the vendor and he's nowhere to be seen. Should that be a separate issue haha?

**Comment by Cheelout** [comment]
Not related to the core. Will be handled offline.
